packages/wasm-edit/src/index.js

```diff
@@ -58,18 +58,21 @@ export function editWithAST(
 
   function after(type: string, path: NodePath<*>) {
     if (path.node._deleted === true) {
+      console.log(type, "deleted");
       operations.push({
         kind: "delete",
         node: path.node
       });
       // $FlowIgnore
     } else if (hashNode(nodeBefore) !== hashNode(path.node)) {
+      console.log(type, "different hash");
```

Comment:
btw please let me know if you have a better idea here? I also tried the tree diffing approach but was quite too complicated.